src/components/learner-credit-management/cards/data/utils.js

```diff
@@ -53,7 +53,7 @@ export const isEmailAddressesInputValueValid = ({
   const hasEnoughBalanceForAssigment = remainingBalanceAfterAssignment >= 0;
 
   const invalidEmails = learnerEmails.filter((email) => !isEmail(email));
-  const duplicateEmails = learnerEmails.filter((email, index) => learnerEmails.indexOf(email) !== index);
+  const duplicateEmails = learnerEmails.filter((email, index) => learnerEmails.indexOf(email.toLowerCase()) !== index);
```

Comment:
handles verification edge case where the user may enter an uppercase and lower case duplicate email. (TEST@gmail.com, test@gmail.com).
Verification at this point in the code results in the validation error retaining the casing of the error, relaying to the user to remove either the uppercase or lowercase value email address.

Reply:
Chose to resolve it in a separate PR, https://github.com/openedx/frontend-app-admin-portal/pull/1101